src/Blazored.Toast/wwwroot/blazored-toast.css

```
@@ -73,15 +73,13 @@
             line-height: 32px;
         }
 
-        .blazored-toast-body .blazored-toast-header button {
+        .blazored-toast-body .blazored-toast-header button.blazored-toast-close {
```

```suggestion
        .blazored-toast-body .blazored-toast-header .blazored-toast-close {
```

I don't think we need the `button` reference now there is a class assigned to it. I would suggest dropping it.

---

True, fixing